test/unit/view/announcer.spec.js

```diff
@@ -59,26 +59,6 @@ it('should remove the element when unmounting after a timeout', () => {
   expect(getElement('5')).not.toBeTruthy();
 });
 
-it('should not remove the element when unmounting after a timeout if the element does not exist', () => {
```

Comment:
I removed this test in favour of one that catches any issues for all the components